client/src/components/flagViewComponents/DeleteFlagModal.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { deleteFlag } from '../../actions/FlagActions';
+
+const DeleteConfirmationModal = ({ deletingFlag, setDeletingFlag, flagId, history }) => {
+	const dispatch = useDispatch();
+
+	const handleCancel = () => {
+		setDeletingFlag(false);
+	};
+
+	const handleSubmit = () => {
+		dispatch(deleteFlag(flagId, () => history.push('/flags')));
+	};
+
+	return (
+		<div id="modal-container" className={`${deletingFlag ? '' : 'hidden'}`}>
+			<div id="modal">
+				<form className="space-y-8 divide-y divide-gray-200 m-8">
+					<div className="space-y-8 divide-y divide-gray-200 sm:space-y-5 p-6">
+						<div>
```

Comment:
@kaledoux I was just wondering if the large indentations in some of these files is intentional? It looks like there are really large indentations throughout some files (like this one) but not others.

Reply:
Not intentional... 
Not sure but I think it might be the GH parser interpreting VSCode indentations differently? 
On my end, all appears normal:
![Screenshot from 2021-07-12 18-15-35](https://user-images.githubusercontent.com/42515081/125362655-441a0c00-e33d-11eb-91bc-82694453aeeb.png)
I have tabs set to two spaces, so I'm not sure why it's chewing it up so badly!